crates/subspace-farmer-components/src/plotting.rs

```diff
@@ -194,8 +193,11 @@ where
                 expanded_piece.extend(chunk);
                 expanded_piece.extend([0]);
             });
-            let polynomial = kzg.poly(&expanded_piece)?;
-            kzg.commit(&polynomial).map_err(Into::into)
+            let polynomial = kzg
+                .poly(&expanded_piece)
+                .map_err(PlottingError::FailedToCommit)?;
```

Comment:
maybe `FailedToConstructPolynimial`?

Reply:
Polynomial creation is a mandatory step within commitment process, this enum variant covers all reasons why commitment might fail. And BTW both errors are not really ever supposed to happen (we fully control correctness of the input here as you can see a few lines above).

It was a variant containing an enum before, but since new library uses strings it has become a bit less obvious. I might work with upstream to fix that though.